addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14087,7 +14087,21 @@ msgctxt "#36043"
 msgid "No CEC capable TV detected. Disable polling for CEC capable devices?"
 msgstr ""
 
-#empty strings from id 36044 to 36100
+#. Desc of action to be taken when "changing CEC source" (2nd)
```

Comment:
As contextual note I'd use for both new labels:
`#. Name of an action to perform on a specific event, like changing the CEC source away from Kodi`